writer.go

```diff
@@ -148,14 +150,35 @@ func WriteElement(e *dicomio.Encoder, elem *Element) {
 		if elem.UndefinedLength {
 			encodeElementHeader(e, elem.Tag, vr, undefinedLength)
 			writeBasicOffsetTable(e, image.Offsets)
-			for _, image := range image.Frames {
+			for _, image := range image.EncapsulatedFrames {
 				writeRawItem(e, image)
 			}
 			encodeElementHeader(e, dicomtag.SequenceDelimitationItem, "" /*not used*/, 0)
 		} else {
```

Comment:
Changes were made to prevent the writing tests from failing. These take into account the new internal representation of `PixelInfo`